src/util.ts

```diff
@@ -1113,11 +1113,20 @@ export function cloneObject<T>(obj: T): T {
   return JSON.parse(JSON.stringify(obj)) as T;
 }
 
-// For MacOS runners: runs `csrutil status` to determine whether System
-// Integrity Protection is enabled.
-export async function isSipEnabled(
+// The first time this function is called, it runs `csrutil status` to determine
+// whether System Integrity Protection is enabled; and saves the result in an
+// environment variable. Afterwards, simply return the value of the environment
+// variable.
+export async function checkSipEnablement(
   logger: Logger,
 ): Promise<boolean | undefined> {
+  if (
+    process.env[EnvVar.IS_SIP_ENABLED] !== undefined &&
+    process.env[EnvVar.IS_SIP_ENABLED] in ["true", "false"]
```

Comment:
This is a TypeScript gotcha: `in` just looks at keys, e.g. `"true" in {"true": 1, "false": 1}` is true, but `"true" in ["true", "false"]` is false.

```suggestion
    ["true", "false"].includes(process.env[EnvVar.IS_SIP_ENABLED])
```

Reply:
Thanks for the catch!!